src/renderer/components/common/SelectableSignatory/SelectableSignatory.tsx

```diff
@@ -36,18 +36,10 @@ const SelectableSignatory = <T extends any>({
 }: Props<T>) => {
   const address = getAddress(addressProps);
 
-  const { getBalance } = useBalance();
+  const { getLiveBalance } = useBalance();
   const popoverItems = useAddressInfo(address, explorers, true);
 
-  const [balance, setBalance] = useState('');
-
-  useEffect(() => {
-    if (chainId && asset) {
-      getBalance(toAccountId(address), chainId, asset.assetId.toString()).then((b) =>
-        setBalance(transferableAmount(b)),
-      );
-    }
-  }, [chainId, asset, address]);
+  const balance = chainId && asset && getLiveBalance(toAccountId(address), chainId, asset.assetId.toString());
```

Comment:
`getLiveBalance` is react hook, so it can't be conditional
If for some reason `chainId` or `asset` change between renders application will fail with error
We gotta make sure that `asset` and `chainId` are defined before we pass it to `SelectableSignatory `